packages/react/src/profiler.tsx

```diff
@@ -86,16 +82,16 @@ class Profiler extends React.Component<ProfilerProps> {
       const changedProps = Object.keys(updateProps).filter(k => updateProps[k] !== this.props.updateProps[k]);
       if (changedProps.length > 0) {
         const now = timestampInSeconds();
-        // eslint-disable-next-line deprecation/deprecation
-        this._updateSpan = this._mountSpan.startChild({
+        this._updateSpan = startInactiveSpan({
+          name: `<${this.props.name}>`,
+          op: REACT_UPDATE_OP,
+          origin: 'auto.ui.react.profiler',
+          startTimestamp: now,
           data: {
             changedProps,
             'ui.component_name': this.props.name,
           },
-          description: `<${this.props.name}>`,
-          op: REACT_UPDATE_OP,
-          origin: 'auto.ui.react.profiler',
-          startTimestamp: now,
+          parentSpanId: this._mountSpan.spanContext().spanId,
```

Comment:
this will not work anymore going forward - `parentSpanId` is/will be deprecated. Instead, you need to pass a `scope` that has the correct span as active span! 😬 But you can basically do this:

```js
const scope = new Scope();
scope.setSpan(this._mountSpan);
startInactiveSpan({ scope, ... });
```

Reply:
dang, good to know. Fixed with 64a14b617187505e29c3cbe394f796d7b8685405

Reply:
The way to go is https://github.com/getsentry/sentry-javascript/pull/10195, which Luca just added for exactly this case :D